zookeeper-server/src/main/java/org/apache/zookeeper/server/quorum/QuorumCnxManager.java

```diff
@@ -202,78 +201,81 @@
             }
         }
 
-        static public InitialMessage parse(Long protocolVersion, DataInputStream din)
-            throws InitialMessageException, IOException {
+        public static InitialMessage parse(
+                Long protocolVersion,
+                DataInputStream din
+        ) throws InitialMessageException, IOException {
             Long sid;
 
             if (protocolVersion != PROTOCOL_VERSION) {
-                throw new InitialMessageException(
-                        "Got unrecognized protocol version %s", protocolVersion);
+                throw new InitialMessageException("Got unrecognized protocol version %s", protocolVersion);
             }
 
             sid = din.readLong();
 
             int remaining = din.readInt();
             if (remaining <= 0 || remaining > maxBuffer) {
-                throw new InitialMessageException(
-                        "Unreasonable buffer length: %s", remaining);
+                throw new InitialMessageException("Unreasonable buffer length: %s", remaining);
             }
 
             byte[] b = new byte[remaining];
-            int num_read = din.read(b);
+            int numRead = din.read(b);
```

Comment:
I totally agree!
I have talked about this in the ML, we should not change variable names and method signatures.
It is dangerous, especially for pending patches waiting for a merge

Reply:
I disagree (slightly). The more standard naming style is easier to read and a one-time cost on open PRs (which will always exist) and on private patches doesn't seem too bad. It would be different if private repo owners intended to opt out of the change (by reverting the patch when merging) but I don't see why that would be preferable.

Reply:
@enixon I think my main argument on proposing of doing the naming changes as a future pull request instead of as part of this pull request is to ensure this pull request does not contain any semantic changes, make both this and the future pull request easier to review and evaluate. That approach has the down side of having to do another (or other sets of) pull requests for enabling more check styles rules, but I feel the benefit of staging the work out weight the cost of invalidate any PR or causing internal private code rebase multiple times, as each future pull request will be well scoped, and the amortized cost would likely be low for rebasing etc.